sites/chaines-tv.orange.fr/chaines-tv.orange.fr.config.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
       const start = parseStart(item)
       const stop = parseStop(item, start)
       programs.push({
-        title: item.season?.serie?.title ? item.season.serie.title : item.title,
+        title: item?.season?.serie?.title ? item.title : item?.season?.serie?.title,

```

Comment:
I don't understand why check if there is an `item?.season?.serie?.title` and then use `item.title` instead.

If you just want to output the title of the series along with the title of the episode, you can use the `subtitle` property instead:

```suggestion
        title: item.title,
        subTitle: item?.season?.serie?.title,
```

Here is a complete list of available properties: https://github.com/freearhey/epg-grabber?tab=readme-ov-file#program-object

Reply:
The original version of the plugin used item?.season?.serie?.title to gather the name of the program. However, that would only correspond to the TV Show's name, not the title of the episode being shown.

If you want to have "Esprits criminels" and "Tête de liste" as an episode name, it would need to be reversed.